analytics/Dockerfile

```diff
@@ -0,0 +1,125 @@
+# Use the official python3 image based on Debian 11 "Bullseye".
+# https://hub.docker.com/_/python
+
+# The build stage that will be used to deploy to the various environments
+# needs to be called `release` in order to integrate with the repo's
+# top-level Makefile
+FROM python:3-slim AS base
+
+# Install poetry, the package manager.
+# https://python-poetry.org
+RUN pip install --no-cache-dir poetry --upgrade
+
+RUN apt-get update \
+  # Install security updates
+  # https://pythonspeed.com/articles/security-updates-in-docker/
+  && apt-get upgrade --yes \
+  && apt-get install --no-install-recommends --yes \
+  build-essential \
+  libpq-dev \
+  postgresql \
+  wget \
+  # Reduce the image size by clear apt cached lists
+  # Complies with https://github.com/codacy/codacy-hadolint/blob/master/codacy-hadolint/docs/description/DL3009.md
+  && rm -fr /var/lib/apt/lists/* \
+  && rm /etc/ssl/private/ssl-cert-snakeoil.key
+
+# Install gh CLI
+# docs: https://github.com/cli/cli/blob/trunk/docs/install_linux.md
+RUN mkdir -p -m 755 /etc/apt/keyrings && wget -qO- https://cli.github.com/packages/githubcli-archive-keyring.gpg | tee /etc/apt/keyrings/githubcli-archive-keyring.gpg > /dev/null \
+  && chmod go+r /etc/apt/keyrings/githubcli-archive-keyring.gpg \
+  && echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+  && apt update \
+  && apt install gh -y \
+  && gh --version
```

Comment:
This is new versus the api dockerfile

Reply:
Awesome thanks for catching and handling the need to install the `GitHub CLI`